UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/AzureBlobStorageFileNameCheck.cs

```diff
@@ -0,0 +1,112 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.FunctionalTests.Helper;
+using UKHO.ExchangeSetService.API.FunctionalTests.Models;
+
+namespace UKHO.ExchangeSetService.API.FunctionalTests.FunctionalTests.FileNameCheck
+{
+    public class AzureBlobStorageFileNameCheck
+    {
+        private ExchangeSetApiClient ExchangeSetApiClient { get; set; }
+        private TestConfiguration Config { get; set; }
+        private DataHelper DataHelper { get; set; }
+
+        private readonly string sinceDateTime = DateTime.Now.AddDays(-10).ToString("ddd, dd MMM yyyy HH':'mm':'ss 'GMT'", CultureInfo.InvariantCulture);
+        private string EssJwtToken { get; set; }
+        public ProductIdentifierModel ProductIdentifierModel { get; set; }
+
+        [SetUp]
+        public async Task SetupAsync()
+        {
+            Config = new TestConfiguration();
+            ExchangeSetApiClient = new ExchangeSetApiClient(Config.EssBaseAddress);
+            ProductIdentifierModel = new ProductIdentifierModel();
+            AuthTokenProvider authTokenProvider = new AuthTokenProvider();
+            EssJwtToken = await authTokenProvider.GetEssToken();
+            DataHelper = new DataHelper();
+        }
+
+        /// <summary>
+        /// This Test check container created in Azure storage account,
+        /// When Api with Valid RFC1123DateTime is called
+        /// </summary>
+        [Test]
+        public async Task WhenICallTheApiWithAValidRFC1123DateTime_ThenValidFileNameIsPresentInAzureStorage()
+        {
+            var apiResponse = await ExchangeSetApiClient.GetExchangeSetBasedOnDateTimeAsync(sinceDateTime, accessToken: EssJwtToken);
+            Assert.AreEqual(200, (int)apiResponse.StatusCode, $"Incorrect status code is returned {apiResponse.StatusCode}, instead of the expected 200.");
+
+            bool checkFileNameExistInContainer = await AzureBlobStorageCheck.CheckIfFileNameExist(Config.EssStorageAccountConnectionString, apiResponse);
+            Assert.IsTrue(checkFileNameExistInContainer);
+        }
+
+        [Test]
+        public async Task WhenICallTheApiWithAValidProductIdentifiers_ThenValidFileNameIsPresentInAzureStorage()
+        {
+            var apiResponse = await ExchangeSetApiClient.GetProductIdentifiresDataAsync(DataHelper.GetProductIdentifierData(), accessToken: EssJwtToken);
```

Comment:
There is a typo on ExchangeSetApiClient.GetProductIdentifiresDataAsync which should be ExchangeSetApiClient.GetProductIdentifiersDataAsync. (Identifiers)

I realise the issue is on the ExchangeSetApiClient.cs file but wanted to raise it.

Reply:
Thanks for the comment.
Have corrected the typo in the ExchangeSetApiClient.cs file and all related Test cases.